cmd/launch.go

```diff
@@ -312,7 +314,14 @@ func runLaunch(cmdCtx *cmdctx.CmdContext) error {
 		}
 	}
 
-	fmt.Printf("Created app %s in organization %s\n", cmdCtx.AppName, org.Slug)
+	out := iostreams.FromContext(ctx).Out
+
+	adminLink := fmt.Sprintf("https://fly.io/apps/%s", appName)
+	appLink := fmt.Sprintf("%s.fly.dev", appName)
+	links := fmt.Sprintf("Admin URL: %s\nURL: %s \nApp ID: %s \n", adminLink, appLink, cmdCtx.AppName)
+
+	fmt.Fprintln(out, colorize.Yellow(aurora.Underline("App Created\n").String()))
```

Comment:
One reason we have `colorize` is so we can control whether ascii codes are used in the output or not. So, we might want `colorize.Underline` here to work instead of importing aurora directly.